.gitignore

```diff
@@ -6,32 +6,18 @@ lerna-debug.log
 
 coverage
 .DS_Store
-dist
-build/
-website/.cache
 tsconfig.tsbuildinfo
 
 .idea/
 
 .cache-test
-/build-dist/
-
-*.js
 
-!jest-*.js
-!dist-prepare.js
+/build/
+/build-dist/
 
 /mitm/socket/connect
-!mitm/install.js
 /mitm/socket/build/
-/.mitm-ca
```

Comment:
shouldn't this one still be here?

Reply:
Yes. I started the new .secret-agent data dir as part of this same PR, then split.

Will push a fix.